src/Nethermind/Nethermind.TxPool/TxPoolConfig.cs

```diff
@@ -14,6 +14,7 @@ public class TxPoolConfig : ITxPoolConfig
                                                                         // every day about 21600 blobs will be included (7200 blocks per day * 3 blob target)
         public int BlobCacheSize { get; set; } = 256;
         public int InMemoryBlobPoolSize { get; set; } = 512; // it is used when persistent pool is disabled
+        public bool BlobReorgsSupportEnabled { get; set; } = true;
```

Comment:
What if we merge it even now, but with this flag false for now?

Reply:
Should we make the actual logic based on this one && if chain supports blobs (or will support in the future)

Reply:
It is already used only when all `BlobReorgsSupportEnabled`, `BlobSupportEnabled` and `PersistentBlobStorageEnabled` are true, so as blob support and persistent storage are false for default now, we can merge with reorgs = true, it will be disabled anyway
https://github.com/NethermindEth/nethermind/blob/dcbb3d35d490dab70f972025dc57d051b0230271/src/Nethermind/Nethermind.TxPool/TxPool.cs#L97

Reply:
Two things:
1. Maybe better to collapse all those 3 config items into one enum like:

`BlobsSupport: { Disabled, InMemory, Storage, StorageWithReorgs } ` ?

2. Should we also query chainspec to see if 4844 is ever enabled on it and if not disable blobs?